.packit.yml

```diff
@@ -7,10 +6,7 @@ actions:
     - git config --global user.name 'nobody'
     - git config --global user.email '<>'
     - tools/webpack-jumpstart --wait --rebase
-    # the upstream git spec file has unresolved macros which are normally put
-    # in at `make dist` time; they are not important for CI
-    # also, build optional packages for CentOS 8
-    - sh -c 'sed "s/%{npm-version:.*}/0/; s/build_optional 0/build_optional 1/" tools/cockpit.spec > cockpit.spec'
+    - tools/create-spec --version 0 --build-all -o cockpit.spec tools/cockpit.spec.in
```

Comment:
I like the build-all flag, and how you plumb it through the .spec. Thanks!

I just wonder why we have to set "version 0" here. Wouldn't that break the actual version in the .spec, the rpms, and thus produce unsatisfiable dependencies? We e.g. have some minimum bridge version dependency. Can't we figure this out from `git describe` or the built `/.tarball` file in the dist tar?

Reply:
We have to set a version because we need to stick something in there.  packit's "fix spec" step will erase whatever we put here, and fill in its version number anyway.

Reply:
For the record `"$(git describe | tr '-' '.')"`

Reply:
Right, the current code even does that as well (sorry for missing that). Could you please add a comment there like "packit srpm updates the version to the tag name" or so?